pkg/clients/clients.go

```diff
@@ -201,8 +203,17 @@ func NewClients(configPath, clusterName, namespace string) *Clients {
 	c.PipelineRunClient = cs.TektonV1beta1().PipelineRuns(namespace)
 	c.PipelineResourceClient = rcs.TektonV1alpha1().PipelineResources(namespace)
 	c.ConditionClient = cs.TektonV1alpha1().Conditions(namespace)
-	c = initTestingFramework(c)
-	return c
+	c, err = initTestingFramework(c)
+
+	// When we don't subscribe to Operator initTestFramework will throw error waiting for the condition (config to be present on default namespace)
+	if err != nil {
+		if strings.Contains(strings.ToLower(os.Getenv("INSTALL_OPERATOR")), "y") {
```

Comment:
1. I don't like this part... if I make a typo and set `INSTALL_OPERATOR=noy`, it will still fail with confusing message.
2. Shouldn't this be handled directly in `initTestingFramework` so that no error is returned?